packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -39,6 +43,15 @@ import { styles } from './mgt-file-list-css';
 import { strings } from './strings';
 import { MgtFile } from '../mgt-file/mgt-file';
 
+/**
+ * An interface used to extend CacheItem file for CacheFileList.
+ *
+ * @interface CacheFileList
+ */
+interface CacheFileList extends CacheItem {
```

Comment:
We should avoid repeating the same interface as is in `graph.files.ts`. An alternative could be to implement and export a clearFilesCache function in `graph.files.ts` and import and use it in the `reload` method below.